lib/liquid/block.rb

```diff
@@ -35,17 +35,17 @@ def warnings
       all_warnings
     end
 
-    def unknown_tag(tag, params, tokens)
+    def unknown_tag(tag, _params, _tokens)
```

Comment:
I kinda like it, I think it makes the code easier to understand. Will leave it for now.